lailib/image/crop.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from lailib.image.binarize import otsu_thresh
+
+def crop_boundary_and_padding(im, padding=0, binarized=None):
+    '''
+    crop and padding an image, with out padding the cropped image should not have any empty columns
+    or rows on each end of both axises
```

Comment:
better use case description:
crop and padding objects and text imgs with pure black border, then padding...